README.md

```diff
@@ -12,4 +13,38 @@ ask questions.
 - Ask what type of reporting is needed
 - Are there varying level of access?
 
+## June 14, 2022
```

Comment:
Can you add a checkmark right next to 2022? 
(I couldn't find anything to comment lol)